etl_pipeline/pipeline_runner.py

```
@@ -377,3 +390,72 @@ def get_pipeline_summary(self) -> Dict[str, Any]:
+            # Initialize due diligence manager
+            self._initialize_due_diligence_manager()
+            
+            if not self.due_diligence_manager:
+                logger.error("Due diligence manager not initialized")
+                return False
+            
+            # Process due diligence
+            self._process_due_diligence()
+            
+            logger.info("Due diligence processing completed successfully")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)